Packet++/header/TLVData.h

```diff
@@ -305,12 +305,12 @@ namespace pcpp
 		 * @param[in] tlvDataBasePtr A pointer to the TLV data byte stream
 		 * @param[in] tlvDataLen The TLV data byte stream length
 		 * @return An instance of type TLVRecordType that wraps the record following the record given as input. If the
-		 * input record.isNull() is true or if the next record is out of bounds of the byte stream, a logical NULL
+		 * input record.isNull() is true or if the next record is out of bounds of the byte stream, a logical nullptr
```

Comment:
ditto: replace `logical nullptr` to `logical null` here and in other changes in this file